templates/zerver/help/require-topics.md

```diff
@@ -21,7 +21,7 @@ specified topic.
 ## Stream messages without a topic
 
 If a user sends a message without a topic, the message's topic is
-displayed as **(no topic)**.
+displayed as **general chat**.
 
```

Comment:
For consistency with other articles, let's use different formatting here. Also tweaking the wording:

> is set to "general chat".